docker/hive/Dockerfile

```diff
@@ -0,0 +1,56 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM centos:centos7 as hive-base
+
+RUN yum -y install wget java-1.8.0-openjdk maven
+
+RUN wget http://ftp.jaist.ac.jp/pub/apache/hadoop/common/hadoop-3.2.0/hadoop-3.2.0.tar.gz
+RUN tar xvf hadoop-3.2.0.tar.gz
+RUN mv hadoop-3.2.0 /opt/hadoop
+RUN rm -rf /opt/hadoop/share/doc
+
+RUN wget http://ftp.jaist.ac.jp/pub/apache/hive/hive-3.1.1/apache-hive-3.1.1-bin.tar.gz
+RUN tar xvf apache-hive-3.1.1-bin.tar.gz
+RUN mv apache-hive-3.1.1-bin /opt/hive
+
+RUN wget http://ftp.yz.yamagata-u.ac.jp/pub/network/apache//db/derby/db-derby-10.14.2.0/db-derby-10.14.2.0-bin.tar.gz
```

Comment:
"/"  in the url is duplicated. However, it seems not to modify.

Reply:
Thank you.
I fixed it.

Reply:
Thank you.